WORKING_GROUPS.md

```diff
@@ -0,0 +1,213 @@
+
+- **Coordination**: https://github.com/ipfs/pm/blob/master/GOLANG_CORE_DEV_MGMT.md
+- **Captain**: [Jeromy Johnson](https://github.com/whyrusleeping)
+
+Develop go-ipfs.
+
+**Responsibilities include**:
+- Create the Server and reference implementation of the IPFS Protocol.
+
+#### IPFS GUI
+
+- **Coordination**: https://github.com/ipfs/ipfs-gui
+- **Captain**: [Oli Evans](https://github.com/olizilla)
+
+Making IPFS GUIs simple, accessible, reusable and beautiful.
+
+**Responsibilities include**:
+- Fight complexity at every step.
+- IPFS must be usable and comprehensible for everyone.
+- Publish and promote this work. Make doing the right thing the easiest thing.
+- Demonstrate the nature of the system with effortless, coherent, and compelling interfaces.
+
+#### IPFS Infrastructure
+
+- **Coordination**: http://github.com/ipfs/infrastructure
+- **Captain**: [Erin Fahy](https://github.com/eefahy)
+
+Tools and systems for the IPFS community.
+
+**Responsibilities include**:
+- Manage and maintain the IPFS HTTP Gateway
+- Manage and maintain the IPFS Bootstrapper nodes
+- Manage and maintain the IPFS pinning service, pinbot
+- Monitor services and hosts used by the IPFS Dev teams
+- Provide wisdom to other users that want to host IPFS nodes
+
+#### Dev Team Enablement (QA, Testing, Support)
+
+- **Coordination**: https://github.com/ipfs/testing
+- **Captain**: [Victor Bjelkholm](https://github.com/VictorBjelkholm)
+
+The QA, Testing and Dev Team Enablement Working Group focuses on building developer tools, CI and automated maintenance tasks to improve the Developer Experience of the many IPFS (and related) projects.
+
+**Responsibilities include**:
+
+- Enable developers to develop faster by reducing friction from dev.
+- Upgrade and build dev tools (i.e AEgir and gx).
+- Create a sound testing infrastructure for all the IPFS projects.
+- Set the guidelines for writing, maintaining and managing codebases, test suites and contributions.
+- Define process and help achieve high quality Sustainable Open Source Modules.
+- Automating tasks for things such as: release cycle, ensuring code quality, activity reports.
+
+#### Community, Events & Evangelism
+
+> This Working Group hasn't been formed yet.
+
+- **Coordination**: https://github.com/ipfs/community
+- **Captain**: ??? 
+
+The Community, Events & Evangelism Working Group is responsible for building a environment that fosters collaboration, innovation and inclusiveness for the IPFS community.
+
+**Responsibilities include**:
+
+- Organize Meetups in multiple cities.
+- Help community members organize meetups for their own communities.
+- Respond to requests for IPFS presence in conferences.
+- Managing the official Social Media channels.
+- Publish regular updates to the community (blog posts and newsletters).
+
+#### Documentation, Education and Websites
+
+> This Working Group hasn't been formed yet.
+
+- **Coordination**: https://github.com/ipfs/docs
+- **Captain**: ???
```

Comment:
I was also unsure if I would leave this one around. Again, truth is that there has been work and even structure, but since Q3, we no longer have OKRs for Documentation. @Mr0grog was there a change of direction? Is the Docs front finished or going to be pursued in a completely different way?

Reply:
Ugh, sorry for missing this and not replying.

> was there a change of direction? Is the Docs front finished or going to be pursued in a completely different way?

I don’t know. The specific projects I was contracted to do are done and I’ll be gone in a week (after, obviously, an absurdly late delivery). I did not do Q3 OKRs because I’m not here for all of Q3 and nobody asked me to, so it seemed like it would be setting up expectations I could not fulfill. I think/hope the thread of this work is going to be picked up by @meiqimichelle and/or @mikeal, but I really can’t speak for future directions.

Reply:
Hi all! I just finished writing up my thoughts regarding the docs work here: https://github.com/ipfs/user-research/tree/master/distributed-kit . Long story short, setting up and staffing a structure to make sure @Mr0grog 's work continues would address our biggest usability problem, and our biggest blocker to user success, in the IPFS ecosystem. @mikeal and IPFS leadership (not sure exactly who, for this convo) should connect and align on how to staff next steps on this work. 

I know @diasdavid and @flyingzumwalt are short on time right now. Perhaps I'll reach out to @mikeal first to get his thoughts, and then broaden the convo with a proposal rather than waiting for a time when all are available.